pkgs/development/python-modules/radon/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+# Python deps
+, mando
+, colorama
+, pytest-mock
+}:
+
+buildPythonPackage rec {
+  pname = "radon";
+  version = "6.0.1";
```

Comment:
```suggestion
  version = "6.0.1";
  pyproject = true;
```

Reply:
Hm, with `pyproject = true;` and adding `poetry-core` to `nativeBuildInputs` the `result/bin` is not created at all. Am I missing something?

Reply:
This is probably an upstream bug.
How about applying this commit as a patch?
https://github.com/rubik/radon/pull/250/commits/d88d9f5651aab064b5535e3b33d923771dcceeea